packages/transformer/src/IModelExporter.ts

```diff
@@ -675,42 +682,10 @@ export class IModelExporter {
     }
   }
 
-  /** Returns `true` if the specified ElementAspect should be exported or `false` if if should be excluded. */
-  private shouldExportElementAspect(aspect: ElementAspect): boolean {
-    for (const excludedElementAspectClass of this._excludedElementAspectClasses) {
-      if (aspect instanceof excludedElementAspectClass) {
-        Logger.logInfo(loggerCategory, `Excluded ElementAspect by class: ${aspect.classFullName}`);
-        return false;
-      }
-    }
-    // ElementAspect has passed standard exclusion rules, now give handler a chance to accept/reject
-    return this.handler.shouldExportElementAspect(aspect);
-  }
-
-  /** Export ElementAspects from the specified element from the source iModel. */
-  private async exportElementAspects(elementId: Id64String): Promise<void> {
-    const _uniqueAspects = await Promise.all(this.sourceDb.elements
-      ._queryAspects(elementId, ElementUniqueAspect.classFullName, this._excludedElementAspectClassFullNames)
-      .filter((a) => this.shouldExportElementAspect(a))
-      .map(async (uniqueAspect: ElementUniqueAspect) => {
-        const isInsertChange = this._sourceDbChanges?.aspect.insertIds.has(uniqueAspect.id) ?? false;
-        const isUpdateChange = this._sourceDbChanges?.aspect.updateIds.has(uniqueAspect.id) ?? false;
-        const doExport = this._sourceDbChanges === undefined || isInsertChange || isUpdateChange;
-        if (doExport) {
-          const isKnownUpdate = this._sourceDbChanges ? isUpdateChange : undefined;
-          this.handler.onExportElementUniqueAspect(uniqueAspect, isKnownUpdate);
-          await this.trackProgress();
-        }
-      }));
-
-    const multiAspects = this.sourceDb.elements
-      ._queryAspects(elementId, ElementMultiAspect.classFullName, this._excludedElementAspectClassFullNames)
-      .filter((a) => this.shouldExportElementAspect(a));
-
-    if (multiAspects.length > 0) {
-      this.handler.onExportElementMultiAspects(multiAspects);
-      return this.trackProgress();
-    }
+  /** Exports all aspects present in the iModel.
+   */
+  public async exportAllAspects(): Promise<void> {
```

Comment:
this adds a very confusing new beta API, that by default doesn't do anything.
I am not sure how best to redesign the API, maybe as `onAllAspectsReady`.

For now, I would prefer we make it a private method that the IModelTransformer class is aware of and uses.

Reply:
Changed it as you wanted.